src/main/kotlin/xyz/atrius/waystones/data/config/Localization.kt

```diff
@@ -10,14 +10,13 @@ import java.text.MessageFormat
 import java.util.*
 import kotlin.collections.HashMap
 
-class Localization(val plugin: KotlinPlugin, localeTag: String) {
-    private var configFile = File(plugin.dataFolder, "locale-$localeTag.yml")
+class Localization(private val plugin: KotlinPlugin, private val locale: Locale) {
```

Comment:
We should probably be referencing this directly from the configuration property to ensure we are pulling the most recent locale. Reloading should probably reference the config property as well.